services/src/main/java/org/keycloak/services/resources/admin/AuthenticationManagementResource.java

```diff
@@ -599,7 +648,70 @@ public void updateExecutions(@PathParam("flowAlias") String flowAlias, Authentic
             model.setRequirement(AuthenticationExecutionModel.Requirement.valueOf(rep.getRequirement()));
             realm.updateAuthenticatorExecution(model);
             adminEvent.operation(OperationType.UPDATE).resource(ResourceType.AUTH_EXECUTION).resourcePath(session.getContext().getUri()).representation(rep).success();
+            //return;
```

Comment:
Maybe remove this comment?

Reply:
I left it there deliberately. It is now removed